ghcide.cabal

```diff
@@ -170,7 +170,10 @@ executable ghcide
         hslogger,
         base == 4.*,
         binary,
+        base16-bytestring >=0.1.1 && <0.2,
+        bytestring,
         containers,
+        cryptohash-sha1 >=0.11.100 && <0.12,
```

Comment:
not that it really matters here but `sha1` seems like an odd choice. Why not `sha256`?

Reply:
No real reason, I just reused what `hie-bios` was doing to hash the flags